repository-data/application/src/main/resources/hcm-config/configuration/frontend/cms.yaml

```diff
@@ -19,7 +19,7 @@ definitions:
       - jpeg
       - jpg
       - png
-      max.file.size: 100M
+      max.file.size: 1000M
```

Comment:
😃

Reply:
I'd actually go with 3k, given that the largest file we know the Statistical Publications guys have (the one they keep outside of GOSS, is 1.4GB).